guide-us-english.tex

```diff
@@ -362,6 +362,10 @@ \subsection{Backlog Prioritization}
 
 \subsection{Backlog Decomposition and Refinement}
 
+In Scrum@Scale, backlog gets pulled by teams.
+In order for backlog from higher level teams to be pulled, it might need to be broken down and understood better.
+This is done in the Backlog Decomposition and Refinement component of the PO cycle.
+
```

Comment:
Needs more work. But this version isn't a bad draft. I'd like to see a mention of the relationship of the SoS backlog and the team backlog. This version may give the impression it's just about the Scrum Teams in an SoS structure.